cmd/oras/root/repo/tags.go

```diff
@@ -115,10 +131,17 @@ func showTags(cmd *cobra.Command, opts *showTagsOptions) error {
 	}
 	err = finder.Tags(ctx, opts.last, func(tags []string) error {
 		for _, tag := range tags {
+			if targetPrefix != "" {
+				if scopedTag, ok := strings.CutPrefix(tag, targetPrefix); ok {
+					tag = scopedTag
+				} else {
+					continue
+				}
+			}
```

Comment:
nit: Might be helpful to add an empty line between these if statements to make it more clear.

Reply:
Added empty lines.